code/__HELPERS/text.dm

```diff
@@ -393,6 +393,33 @@ proc/checkhtml(var/t)
 	if(parts.len==2)
 		. += ".[parts[2]]"
 
+
+/**
+ * Formats unites with their suffixes
+ * Should be good for J, W, and stuff
+ */
+var/list/unit_suffixes = list("", "K", "M", "G", "T", "P", "E", "Z", "Y")
```

Comment:
Alright time to fix this.

That `K` has to be lower case.

This is very important.